app/src/views/auth/set_up_screen.tsx

```diff
@@ -88,7 +88,7 @@ export default function SetUsernameScreen({
           />
         </ThemedView>
 
-        {errorMessage && (
+        {Boolean(errorMessage) && (
```

Comment:
Same here.

Reply:
Replied in forgot_password_screen.tsx comments